website/conferences/views.py

```diff
@@ -215,6 +215,14 @@ def conference_view(**kwargs):
         )
         projects = Node.find(query)
         submissions = projects.count()
+        downloads_list = [
```

Comment:
So more popular conferences are more likely to have more projects and also more people looking at them, yes? If we tie up a worker for a few seconds each time on a big conference, it seems like the wrong confluence of popularity and slowness. Might want to think about a caching method or some kind of async way of doing this.